doc/reference/reference_lua/datetime.rst

```diff
@@ -1080,10 +1110,68 @@ This section describes how the ``datetime`` module supports leap seconds:
         - 1970-01-01T00:01:00Z
         ...
 
+.. _timezone:
+
+Time zones
+----------
+
+Full support has been added since :doc:`2.11.0 </release/2.11.0>`.
+
+Tarantool uses `Time Zone Database <https://www.iana.org/time-zones>`__
+(also known as Olson database and supported by IANA) for timezone support.
+You can use the Lua module :ref:`tarantool <tarantool-module>` to get a used version of ``tzdata``.
+
+Every datetime object has three fields that represents timezone support:
```

Comment:
```suggestion
Every datetime object has three fields that represent timezone support:
```

Or we can make it shorter: `fields that represent timezone support -> fields for timezone support`.